### PR TITLE
procexp: new verb

### PR DESCRIPTION
**Pull request by rhabacker**
This pull request adds [process explorer](https://learn.microsoft.com/de-de/sysinternals/downloads/process-explorer) to winetricks.

### CODE REVIEW

File: src/winetricks [hunk 1]
```diff
@@ -16895,6 +16895,21 @@ _EOF_
...
 
+w_metadata procexp apps \
+    title="Process Explorer" \
+    publisher="Steve P. Miller" \
+    year="2006" \
+    media="download" \
+
+load_procexp()
+{
+    w_download https://download.sysinternals.com/files/ProcessExplorer.zip
```
Comment (JonnyTech) [inline src/winetricks:16906]: You can use direct links from the `live` site:
https://live.sysinternals.com/procexp.exe
or
https://live.sysinternals.com/procexp64.exe

Comment (austin987) [reply src/winetricks:16906]: Please add a checksum here.

File: src/winetricks [hunk 2]
```diff
@@ -16895,6 +16895,21 @@ _EOF_
...
+    title="Process Explorer" \
+    publisher="Steve P. Miller" \
+    year="2006" \
+    media="download" \
+
+load_procexp()
+{
+    w_download https://download.sysinternals.com/files/ProcessExplorer.zip c50bddaaacb26c5654f845962f9ee34db6ce26b62f94a03bb59f3b5a6eea1922
+    w_try_unzip "${W_TMP}" "${W_CACHE}"/procexp/ProcessExplorer.zip
+    w_try cp "${W_TMP}"/procexp*.exe "${W_DRIVE_C}/windows"
```
Comment (austin987) [inline src/winetricks:16908]: This will install the 64bit exe in a 32-bit prefix.

Also, please use `${W_WINDIR_UNIX}` here.

Comment (rhabacker) [reply src/winetricks]: The issues has been fixed - thanks for reviewing.

**Pull request merged**